apps/spruce/src/pages/adminSettings/tabs/GeneralTab/transformers.ts

```diff
@@ -0,0 +1,21 @@
+import { AdminSettingsTabRoutes } from "constants/routes";
+import { FormToGqlFunction, GqlToFormFunction } from "../types";
+
+type Tab = AdminSettingsTabRoutes.General;
+
+export const gqlToForm = ((data) => {
+  if (!data) return null;
+
+  const { announcements } = data;
```

Comment:
Check out the comment I left in the EVG [PR](https://github.com/evergreen-ci/evergreen/pull/8970). We may want to replace announcements `banner` and `bannerTheme`